egs/wsj/s5/utils/perturb_data_dir_speed.sh

```diff
@@ -76,7 +76,8 @@ if [ -f $srcdir/segments ]; then
   utils/apply_map.pl -f 1 $destdir/reco_map <$srcdir/wav.scp | sed 's/| *$/ |/' | \
     awk -v factor=$factor \
         '{wid=$1; $1=""; if ($NF=="|") {print wid $_ " sox -t wav - -t wav - speed " factor " |"}
-          else {print wid " sox -t wav" $_ " -t wav - speed " factor " |"}}' > $destdir/wav.scp
+          else if (match($0, /:[0-9]+$/)) {print wid " wav-copy" $_ " | sox -t wav - -t wav - speed " factor " |" } 
```

Comment:
it might be a good to add a comment  a few lines above, explaining why there need to be 3 cases.